README.yaml

```diff
@@ -114,6 +114,47 @@ usage: |-
         max-number-of-prs: 5
   ```
 
+  ### Customize Pull Request labels, title and body
+
+  ```yaml
+    # ...
+    - name: Update Atmos Components
+      uses: cloudposse/github-action-atmos-component-updater@v1
+      with:
+        github-access-token: ${{ secrets.GITHUB_TOKEN }}
+        max-number-of-prs: 5
+        pr-title: 'Update Atmos Component \`{{ component_name }}\` to {{ new_version }}'
+        pr-body: |
+          ## what
+          Component \`{{ component_name }}\` was updated [{{ old_version }}]({{ old_version_link }}) → [{{ old_version }}]({{ old_version_link }}).
+
+          ## references
+          - [{{ source_name }}]({{ source_link }})
+        pr-labels: |
+          component-update
+          automated
+          atmos
+  ```
+  
+  **Note** that backtick ` symbols are escaped.
```

Comment:
```suggestion
 **IMPORTANT:** The backtick symbols must be escaped in the GitHub Action parameters. This is because GitHub evaluates whatever is in the backticks and it will render as an empty string.
```